### PR TITLE
Ports the salute emote sound from CM-SS13

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Ported the salute audio from the following commit: https://github.com/cmss13-devs/cmss13/commit/7fc3e306c13850c0e5a77175870829d039b105d2
Suggested by https://www.paradisestation.org/forum/topic/24315-suggestion-a-boots-clack-sound-when-someone-uses-the-salute-emote/#comment-173945

The salute sound plays only if you are wearing jackboots (sandals version included), combat boots and laceup/dress shoes.

**EDIT**
Ducky and clown shoes now play a sound too ('toysqueak1.ogg') when saluting.
## Why It's Good For The Game

Having a sound increases the impact of a salute, which is good for immersion (and muh military roleplay!).
## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/77684085/26816b4f-1ac2-4d33-a163-86e68b359319

## Testing

- Tested the emote with different kinds of shoes.
## Changelog
:cl:
soundadd: The salute emote now plays a sound if you are wearing jackboots/dress shoes or clown/ducky shoes
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 1]
```diff
@@ -216,7 +216,14 @@
 	key_third_person = "salutes"
 	message = "salutes."
 	message_param = "salutes to %t."
+	emote_type = EMOTE_AUDIBLE
 	hands_use_check = TRUE
+	audio_cooldown = 3 SECONDS
+
+/datum/emote/living/carbon/human/salute/get_sound(mob/living/user)
+	var/mob/living/carbon/human/H = user
+	if(istype(H.shoes, /obj/item/clothing/shoes/jackboots) || istype(H.shoes, /obj/item/clothing/shoes/combat) || istype(H.shoes, /obj/item/clothing/shoes/centcom))
```
Comment (Contrabang) [inline code/modules/mob/living/carbon/human/human_emote.dm:225]: can you make a list var on the emote, then do is_type_in_list?

Comment (Henri215) [reply code/modules/mob/living/carbon/human/human_emote.dm:225]: Will do

**Comment by spacegeese** [comment]
if saluting while wearing the clown shoes and ducky slippers doesn't make a funny squeak sound, i will die

**Comment by Henri215** [comment]
> if saluting while wearing the clown shoes and ducky slippers doesn't make a funny squeak sound, i will die

Added the toysqueak1.ogg sound for clown shoes and ducky slippers when saluting.

**Review by Burzah — APPROVED** [review]
lgtm

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 2]
```diff
@@ -216,7 +216,20 @@
 	key_third_person = "salutes"
 	message = "salutes."
 	message_param = "salutes to %t."
+	emote_type = EMOTE_AUDIBLE
```
Comment (lewcc) [inline code/modules/mob/living/carbon/human/human_emote.dm:219]: I think this is more of what you meant to do. EMOTE_SOUND will mark it as an audio emote, and hide it from runechat. 

```suggestion
	emote_type = EMOTE_SOUND | EMOTE_VISIBLE
```

**Review by lewcc — COMMENTED** [review]
One little thing, though the rest looks fine.

File: code/modules/mob/living/carbon/human/human_emote.dm (hunk 2 above)
Comment (Henri215) [reply code/modules/mob/living/carbon/human/human_emote.dm:219]: I got a little confused when looking at the types we had, this is what i meant, thanks

File: code/modules/mob/living/carbon/human/human_emote.dm [hunk 3]
```diff
@@ -216,7 +216,20 @@
 	key_third_person = "salutes"
 	message = "salutes."
 	message_param = "salutes to %t."
+	emote_type = EMOTE_SOUND | EMOTE_VISIBLE
```
Comment (Henri215) [inline code/modules/mob/living/carbon/human/human_emote.dm:219]:
```suggestion
```
Let it default to EMOTE_VISIBLE Henri

Comment (Henri215) [reply code/modules/mob/living/carbon/human/human_emote.dm:219]: Oh thank you Henri, you are so smart!

Comment (DGamerL) [reply code/modules/mob/living/carbon/human/human_emote.dm]: Truly the smartest reviewer of the entire codebase

**Review by lewcc — APPROVED** [review]

**Comment by spacegeese** [comment]
incredible

**Review by S34NW — APPROVED** [review]

**Pull request merged**